src/framework/theme/components/helpers.ts

```diff
@@ -25,3 +26,8 @@ export function getElementHeight (el) {
   const marginBottom = parseInt(style.getPropertyValue('margin-bottom'), 10);
   return el.offsetHeight + marginTop + marginBottom;
 }
+
+export function isFirstChange(simpleChanges: SimpleChanges): boolean {
+  const changes: SimpleChange[] = Object.values(simpleChanges);
+  return changes.some((change: SimpleChange) => change.isFirstChange());
```

Comment:
The some() method tests whether at least one element in the array passes the test implemented by the provided function.

Does that mean that we treat the whole changes object as `first` even it could be the second change for other elements except one?